### ISSUE TITLE
Contamination of metadata fields-Geographic location and site where genome was sequenced

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#442]
I have found that while PATRIC had a lot of data with Boston, MA as the geographic location.  As I know that is the site of the Broad Institute, and that the sequencing was done there, I started checking BioSample to see if the data was consistent.  Its not.  Here's what we have:
![screen shot 2015-09-22 at 8 00 33 am](https://cloud.githubusercontent.com/assets/11475316/10017920/65866ad4-6100-11e5-92a3-13e827698acc.png)
Here's what BioSample has:
![screen shot 2015-09-22 at 8 00 07 am](https://cloud.githubusercontent.com/assets/11475316/10017923/6f906b88-6100-11e5-9dbd-3fc078fb7a94.png)
I'm not sure how we fix this, but we need to do it.

**Comment by mshukla1** [comment]
I believe we have the correct metadata and NCBI is missing it. Here is why: 

- This genome, Enterobacter sp. MGH 1, is part of Broad Carbapenem Resistance Project. We received 
metadata directly from Broad as spread sheet, which was curated and incorporated into PATRIC. 
I checked the curated spread sheet, and it does have geographic location as: "Boston, MA, USA" . 
Spreadsheet available in box at: 
 
https://app.box.com/files/0/f/2663838781/1/f_23723024391

- Just to make sure, it was not in the metadata spreadsheet by mistake, I also checked the original white paper for the carbapenem resistance project at Broad. The white paper is titles as: 

https://www.broadinstitute.org/science/projects/gscid/white-papers'

Carbapenem Resistant bacteria
Whole Genome Sequence Analysis of Carbapenem-resistant Enterobacteriaceae Isolated from 3 Boston Area Medical Institutions

Furthermore, in the white paper, it says: 
 
https://www.broadinstitute.org/files/shared/genomebio/Carbapenem_resistant_bacteria_WP.pdf

"In phase I, we will compare the genomic sequences of prospectively sampled carbapenemresistant
enterobacteria in three major medical institutions in Boston, and will monitor
changes in sequenced isolates over a six to nine month period within each institution. We
will also sequence a number of retrospectively collected isolates of carbapenem-resistant
and susceptible strains from each institution for comparative purposes." 
 
Let me know if you find any evidence that suggest it is NOT from one of the Boston area hospitals. 

-Maulik
